.github/workflows/vast.yaml

```diff
@@ -248,13 +248,13 @@ jobs:
             # Zero the cache statistics (but not the configuration options).
             ccache --zero-stats
             ccache --show-config
-            ./configure \
-              --prefix="${PWD}/opt/vast" \
-              --build-dir="$BUILD_DIR" \
-              --package-name="$PACKAGE_NAME" \
-              --with-lsvast \
-              --with-dscat \
-              --with-bundled-caf \
+            cmake -B "$BUILD_DIR" \
+              -DCMAKE_INSTALL_PREFIX="${PWD}/opt/vast" \
```

Comment:
No need to set the prefix here for a relocatable build, instead we should install using `cmake --install "${BUILD_DIR}" --prefix "${PWD}/opt/vast"`.

Reply:
I don't disagree, but this PR is about removing the configure script so I'd like to exactly replicate the current behavior, and then do any refactorings in a separate PR.

Reply:
Fair enough.